contracts/CourtPresaleActivate.sol

```diff
@@ -184,28 +184,29 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
         uint256 contributionTokenAmount = uniswapExchange.ethToTokenSwapInput.value(msg.value)(_minTokens, _deadline);
 
         // buy in presale
-        _buyAndRegisterAsJuror(msg.sender, contributionTokenAmount, contributionToken, _activate);
+        _buyAndActivateAsJuror(msg.sender, contributionTokenAmount, contributionToken, _activate);
     }
 
     function _hasData(bytes memory _data) internal pure returns (bool) {
         return _data.length > 0;
     }
 
-    function _buyAndRegisterAsJuror(address _from, uint256 _amount, ERC20 _token, bool _activate) internal {
+    function _buyAndActivateAsJuror(address _from, uint256 _amount, ERC20 _token, bool _activate) internal {
```

Comment:
As a small thing, but may renaming `_token` to `_contributionToken` would make things clearer from a glance at this function.

Reply:
Definitely! At some point I was confused, and I wrote it! :see_no_evil: